eq.yml

```diff
@@ -819,6 +819,7 @@ jobs:
         - -exc
         - |
           cd eq-survey-runner-deploy
+          tfenv install 0.10.8
```

Comment:
Blocking review so we can remove these installs once [eq-ecs-deploy PR](https://github.com/ONSdigital/eq-ecs-deploy/pull/12) has merged.

Reply:
@dwyeradam I have removed all of the install versions from the `eq-ecs-deploy` modules